DEVELOPMENT.md

```diff
@@ -87,6 +87,23 @@ Maintain the same quality for production and test code.
 
 When writing a Git commit message, follow these [guidelines](https://chris.beams.io/posts/git-commit/).
 
+### Git merge strategy
+
+PRs are usually merged into `master` using the  `rebase and merge` strategy.
+
+It is important to keep commits on feature branches neat, squashing the feature branch as necessary.
```

Comment:
I would either remove this paragraph or move it to the end of this section..

Reply:
Moved the paragraph to the end - I think the context it provides is good to have.